### PR TITLE
added numpy.indices to numpy frontend

### PR DESCRIPTION
**Pull request by lamaran1**

### CODE REVIEW

**Comment by RickSanchezStoic** [comment]
Hi, @lamaran1 can you try fixing the lint errors and the test failures? Thanks!

**Comment by lamaran1** [comment]
@Ric

> Hi, @lamaran1 can you try fixing the lint errors and the test failures? Thanks!

Okay, on it

**Comment by lamaran1** [comment]
> Hi, @lamaran1 can you try fixing the lint errors and the test failures? Thanks!

Hi @RickSanchezStoic, i have been able to fix the lint errors but i cant tell the source of the nightly-test error, on the discord server, it seems it was ignored without solutions. Is this something you could help me check and direct me? thank you.

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_indexing_routines/test_generating_index_arrays.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=False,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        frontend="numpy",
+        fn_tree="numpy.indices",
```
Comment (RickSanchezStoic) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_indexing_routines/test_generating_index_arrays.py:36]: change this to just `indices`

**Review by RickSanchezStoic — CHANGES_REQUESTED** [review]

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_indexing_routines/test_generating_index_arrays.py (hunk 1 above)
Comment (lamaran1) [reply ivy_tests/test_ivy/test_frontends/test_numpy/test_indexing_routines/test_generating_index_arrays.py]: thank you

**Comment by lamaran1** [comment]
Made the change but still getting an error on the run-nightly-tests

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**